stubs/protobuf/METADATA.toml

```
@@ -1,2 +1,2 @@
-version = "3.20.*"
+version = "4.21.*"
```

```suggestion
version = "3.21.*"
```

Sort of a holdover in the meantime

---

(if you like the idea, feel free to commit it and merge - otherwise we can just wait.)

---

Sure, let's do it!